msolve/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM ubuntu:18.04
```

Comment:
Now that the dockerfiles that define the image we will build-test-deploy on Daint are in the `ci` path, this isn't needed.